app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -136,6 +137,10 @@ def releases
 
     api :POST, "/organizations/:id/autoattach_subscriptions", N_("Auto-attach available subscriptions to all hosts within an organization. Asynchronous operation."), :deprecated => true
     def autoattach_subscriptions
```

Comment:
This API was deprecated a couple years back and it looks like it's not being used anywhere. In the spirit of the recent PR which removed other deprecations I think this one is a good candidate for the same. Sound good @sseelam2  @jlsherrill @johnpmitsch ?

Reply:
It sounds like it is a good candidate for removal, just would need to make sure the UI and hammer is not using it or that usage is modified.

I'm surprised there was no warning that went along with the apipie deprecation (though I'm not sure if those are even showing after the community discussion). But I see it has been 4 years since it has been added so is probably safe to remove.

Reply:
Totally agree. @sseelam2 I think we can remove this method, its tests, and its route in config/routes